boards/waspmote-pro/Makefile.include

```diff
@@ -43,14 +43,5 @@ ifeq ($(PROGRAMMER), stk500v1)
     export PROGRAMMER_FLAGS = -P $(PORT) -b 115200
 endif
 
-# define build specific options
-export CFLAGS_CPU   = -mmcu=atmega1281  $(CFLAGS_FPU)
-export CFLAGS_LINK  = -fno-builtin -fshort-enums
-export CFLAGS_DBG  ?= -ggdb -g3
-export CFLAGS_OPT  ?= -Os
-
-export CFLAGS += $(CFLAGS_CPU) $(CFLAGS_LINK) $(CFLAGS_DBG) $(CFLAGS_OPT)
-export ASFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG)
-export LINKFLAGS += $(CFLAGS_CPU) $(CFLAGS_DBG) $(CFLAGS_OPT) -e reset_handler
```

Comment:
OK, got it. Then it should belong to a separated commit, since the naming I proposed won't work with this board, especially because it's being moved to cpu/atmega_common.